src/acq/channel.h

```diff
@@ -20,8 +20,8 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#include "common/error.h"
-#include "common/channel.h"
+#include "../../common/error.h"
+#include "../../common/channel.h"
```

Comment:
You shouldn't need to do this. The include paths should be set up in the CMakeLists.txt file for the project, and if they are not correct then fix the cmake file.